tests/AssetManagerTest/Resolver/AggregateResolverTest.php

```diff
@@ -69,5 +74,21 @@ public function testCollect()
         $this->assertContains('three', $collection);
 
         $this->assertCount(3, $collection);
+
+        /* Tests for interfaces that _don't_ implement the `collect` method. */
+        $resolver    = new AggregateResolver();
```

Comment:
The way it is now, each method has a test method. If I do this, I should do this all over the codebase.